AccountService/Domain/Account.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using Newtonsoft.Json.Converters;
+
+namespace AccountService.Domain
```

Comment:
Ik weet het niet zeker, maar is dit niet gewoon een Entity? Een domain is inclusief repository en alles wat daarbij komt kijken.